contracts/refs/CoreRefV2.sol

```diff
@@ -268,21 +268,27 @@ abstract contract CoreRefV2 is ICoreRefV2, Pausable {
 
     /// @notice hook into the pcv oracle, calls into pcv oracle with delta
     /// if pcv oracle is not set to address 0, and updates the liquid balance
-    function _liquidPcvOracleHook(int256 delta) internal {
+    function _liquidPcvOracleHook(
+        int256 deltaBalance,
+        int256 deltaProfit
+    ) internal {
         IPCVOracle _pcvOracle = pcvOracle();
         if (address(_pcvOracle) != address(0)) {
             /// if any amount of PCV is withdrawn and no gains, delta is negative
-            _pcvOracle.updateLiquidBalance(delta);
+            _pcvOracle.updateLiquidBalance(deltaBalance, deltaProfit);
```

Comment:
As the PCVOracle will possibly change in the future, I wonder if there's a way to make the interface more resilient/less likely to need updating (and so force a redeploy of PCVdeposits)

`pcvOracle.updateBalance(uint8 venueType, uint256 deltaBalance, uint256 deltaProfit)`? 

Rather than specific external methods for `updateLiquidBalance()` or `updateIlliquidBalance()`. Would allow you in the future to add new venueTypes in a new PCVOracle, remove the distinction if you ever wanted to etc

Reply:
but that is a good idea that would be more future proof